.github/workflows/deploy.yaml

```
@@ -0,0 +1,42 @@
+name: Deploy to Firebase
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10
+
+      - name: Installing Dependencencies
+        run: npm install
+
+      - name: Building the project
+        run: npm run build
+
+      - name: Deploy to Firebase
+        uses: w9jds/firebase-action@master
+        with:
+          args: deploy
+        env:
+          # Change your project ID here according to your configuration
+          PROJECT_ID: hoverboard-master
```

Maybe `PROJECT_ID` should be set as a "secret" so this file doesn't have to be edited?

```suggestion
          PROJECT_ID: ${{ secrets.FIREBASE_PROJECT_ID }}
```